core/keyserver/keyserver.go

```diff
@@ -189,13 +189,13 @@ func (s *Server) batchGetUserByRevision(ctx context.Context, sth *tpb.SignedLogR
 		Index:    indexes,
 		Revision: mapRevision,
 	})
-	if err != nil {
+	if st := status.Convert(err); st.Code() != codes.OK {
```

Comment:
Is this preferred over checking err != nil? status upholds that the code is not OK iff err is non-nil.

Reply:
It works regardless of whether err is a Status error or nil. 
Creating it in this way saves us a line of code and scopes `st` to the error block too. 
Not sure what the 100% canonical pattern for doing this is.

Reply:
I think Jon is maybe suggesting passing the error up the stack?